resources/views/service/centreusers/edit.blade.php

```diff
@@ -0,0 +1,147 @@
+@extends('service.layouts.app')
+@section('content')
+
+<div id="container">
+    @include('service.includes.sidebar')
+    <div id="main-content">
+
+        <h1>Edit a Children's Centre Worker</h1>
+
+        <p>Use the form below to amend a Children's Centre Worker's details.</p>
+
+        <!-- ADD NEW POST ROUTE HERE eg. method="POST" action="{{ route('admin.vouchers.storebatch') }}" -->
```

Comment:
Not ready yet?

Reply:
again, in the submit/update handler branch.